language-support/codegen-common/src/main/scala/com/digitalasset/daml/codegen/lf/DefTemplateWithRecord.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.codegen.lf
+
+import com.daml.lf.iface
+
+case class DefTemplateWithRecord[+Type](
+    `type`: iface.Record[Type],
+    template: iface.DefTemplate[Type],
+)
+object DefTemplateWithRecord {
+  type FWT = DefTemplateWithRecord[iface.Type]
```

Comment:
What does FWT stand for?  Is it used?

Reply:
> What does FWT stand for?

Field With Type? Probably @S11001001 can give more context, I just copied this code.

> Is it used?

Yes, six instances.

---

I would probably have a look and see if this needs to be generic.

Reply:
Stands for `Field With Type`; the design was changed so only the `Type` remains in this usage but the many references to the alias were left. I don't know if this particular alias is used but the alias in general is used a lot more than 6 times.

Reply:
I would just extend `FWTLike` here by the way. That will make renaming and find-usages a lot more straightforward should we do that in the future.

Reply:
I'm already working on top of this, I can remove this unnecessary generality but I have changes stacked up on top of this and I would need to refactor this at a later time, is that a problem?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13372 (which builds on top of https://github.com/digital-asset/daml/pull/13370/)